### ISSUE TITLE
System.IO.FileNotFoundException : Could not load file or assembly 'netstandard2.0assembly with Selenium 4.0.0-alpha04 and .NET Core

### ISSUE THREAD

**Issue by moatazeldebsy** [SeleniumHQ/selenium#7932]
## 🐛 Bug Report

I using `.NET Core` with `C#` and I got this error when upgraded to Selenium `4.0.0-alpha04` 

`System.IO.FileNotFoundException : Could not load file or assembly 'netstandard2.0assembly, Version=0.0.0.0, Culture=neutral, PublicKeyToken=null'. The system cannot find the file specified.
`
and the browser didn't open it failed from the beginning. 
## To Reproduce

Detailed steps to reproduce the behavior:

## Expected behavior

The browser should be opened and run the script 

## Test script or set of commands reproducing this issue

```
using NUnit.Framework;
using OpenQA.Selenium;
using OpenQA.Selenium.Chrome;

namespace Selenium4
{
    public class RelativeLocators
    {
        private IWebDriver _driver;

        [SetUp]
        public void Setup()
        {
            _driver = new ChromeDriver();
            _driver.Navigate().GoToUrl("https://the-internet.herokuapp.com/");
            _driver.Manage().Window.Maximize();
        }

        [Test]
        public void ClickLoginLink()
        {
            _driver.FindElement((RelativeBy.
                WithTagName("li")
                .Below(By.LinkText("Forgot Password"))
                .Above(By.LinkText("Frames")))).Click();
            Assert.IsTrue(_driver.Url.Contains("login"));
        }
    }
}
```
## Screenshot
<img width="1320" alt="Screenshot 2020-01-12 at 15 24 50" src="https://user-images.githubusercontent.com/45570191/72220407-a92a5100-3550-11ea-8bd4-4c37b828c1f3.png">

## Environment
OS: OSX 10.14.6
Browser: Chrome
Browser version:  79
Browser Driver version: ChromeDriver 2.37
Language Bindings version: C#

**Comment by BasHamer** [comment]
Encountered this, downgraded to 4.0.0-alpha03 and it got me past this.

**Comment by jesguerra** [comment]
this solution works well

**Comment by moatazeldebsy** [comment]
@BasHamer I know that I was trying the relative locators with `alpha04` which is not in `alpha03`

**Comment by BasHamer** [comment]
Not sure, I can check if desired. I don't think we use relative locators after we found bugs in version selenium version 3. (It did not handle xpath OR properly )

**Comment by aonorgate** [comment]
I also have this issue with alpha04, unable to run any tests with our framework and have had to revert to alpha03 in the meantime.
We are not using any of the new selenium 4.0.0 functions and it was purely the move to alpha04 which cause this issue to manifest.

**Comment by jimevans** [comment]
There was a bug introduced in the build process for the .NET bindings in alpha 4. It’s been fixed in 52b6789. The next release should not have this issue. As usual, I can offer no specific times for a next release.
